DCHdigi/dataFormatExtension/driftChamberHit.yaml

```diff
@@ -41,3 +42,101 @@ datatypes:
     OneToOneRelations:
       - extension::DriftChamberDigi digi    // reference to the digitized hit
       - edm4hep::SimTrackerHit sim // reference to the simulated hit
+
+  extension::DriftChamberDigiV2:
+    Description: "Digitized hit (before tracking) for Drift Chamber v2 (requires data extension)."
+    Author: "A. Tolosa-Delgado, B. Francois, CERN"
+    Members:
+      - uint64_t cellID               // ID of the sensor that created this hit
+      - int32_t type                  // type of the raw data hit
+      - int32_t quality               // quality bit flag of the hit
+      - float time                    // time of the hit [ns]
+      - float eDep                    // energy deposited on the hit [GeV]
+      - float eDepError               // error measured on eDep [GeV]
+      - edm4hep::Vector3d position    // point on the sensitive wire (SW) which is closest to the simhit [mm]
+      - edm4hep::Vector3d directionSW // direction of SW
+      - float distanceToWire          // distance hit-wire [mm]
+      - uint32_t clusterCount         // number of clusters associated to this hit
+      - uint32_t clusterSize          // number of electrons per cluster
```

Comment:
what about renaming those `nCluster` and `nElectronPerCluster` to be more explicit?

Reply:
done